### ISSUE TITLE
Bosco should treat repos in team folder as if they were part of the team

### ISSUE THREAD

**Issue by geophree** [tes/bosco#223]
I ran into a problem where `bosco morning` didn't update one of the repos in my team folder because it had been removed from the github team.  However, `bosco run -d` still used the code from that folder.  I think we should standardize all bosco commands to treat any repos in the team folder as if they were in the team for all commands (we may still want to warn if they aren't in the github team, certainly keep the warn we have in `bosco morning`).

Also, `-r` should work to target any `infra`/`service`/`app`/etc regardless of whether it's currently in your team or not.  You should be able to `bosco restart -r infra-nginx-gateway` regardless of whether `infra-nginx-gateway` is in your team or not, as long as it's a dependency.

**Comment by geophree** [comment]
For the `-r` stuff, I'm fine if it only works in a subdir that is it or has it as a dep.

**Comment by cliftonc** [comment]
Agreed, this has annoyed me too - I'll see if I can get to it this week.
